singularity-eos/eos/eos_spiner_rho_sie.hpp

```diff
@@ -308,9 +354,11 @@ class SpinerEOSDependsRhoSie : public EosBase<SpinerEOSDependsRhoSie> {
   static constexpr const int _n_lambda = 1;
   static constexpr const char *_lambda_names[1] = {"log(rho)"};
   DataStatus memoryStatus_ = DataStatus::Deallocated;
+  TransformData Transform_data_;
+  Transformer transformer_;
 };
-
-inline SpinerEOSDependsRhoSie::SpinerEOSDependsRhoSie(const std::string &filename,
+template <template <class> class TransformerT>
+inline SpinerEOSDependsRhoSie<TransformerT>::SpinerEOSDependsRhoSie(const std::string &filename,
```

Comment:
Uggg... I think we need an MR that combines all of these member function definitions with their declarations in the class. I'm sure you guys don't like having to specify these templates all over the place.

Reply:
It's not 100% necessary, so if you're fine keeping it this way, I'm fine with it too. I just don't think there's any advantage to separating the definition from the declaration now that we're header-only (this is a relic from several years ago when we weren't header only).